src/mmw/js/src/app.js

```diff
@@ -67,6 +73,20 @@ var App = new Marionette.Application({
         }
     },
 
+    getLayersModel: function() {
+        return this.layers;
+    },
+
+    showLayerPicker: function() {
+        if (_.contains(settings.get('map_controls'), 'LayerSelector')) {
```

Comment:
Do you mean remove `LayerSelector` from the `map_controls` setting and have it standalone?